lib/src/int32.dart

```diff
@@ -286,39 +286,40 @@ class Int32 implements IntX {
   }
 
   @override
-  int compareTo(other) {
+  int compareTo(Object? other) {
```

Comment:
If we change `implements Comparable<dynamic>` to `implements Comparable<Object>` could we avoid this?

https://github.com/dart-lang/fixnum/blob/13da5123989a5d91cb59e76793c3923dd8d06e90/lib/src/intx.dart#L8

Reply:
hmm good question ill try